packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -1166,6 +1167,13 @@ mixin SchedulerBinding on BindingBase {
       return true;
     }());
   }
+
+  /// When a scheduling error occurs, resets the relative state of EventLoop.
+  @visibleForTesting
+  void resetEventLoop() {
+    _hasRequestedAnEventLoopCallback = false;
```

Comment:
This is odd though, because the scheduled `Timer.run` would still execute...

Reply:
This flag `_hasRequestedAnEventLoopCallback ` was corrected by the previous test error, as you can see from my previous debugging procedure

https://github.com/flutter/flutter/pull/82781#issuecomment-850257513